OMEdit/OMEditGUI/MainWindow.cpp

```diff
@@ -661,17 +661,20 @@ void MainWindow::openResultFiles(QStringList fileNames)
   }
 }
 
+bool MainWindow::validateIfNeeded(LibraryTreeItem *pLibraryTreeItem)
+{
+  ModelWidget *pModelWidget = pLibraryTreeItem->getModelWidget();
+  return !pModelWidget || pModelWidget->validateText(&pLibraryTreeItem);
+}
+
```

Comment:
Doesn't make sense to have this function in MainWindow.
If its prime purpose is to get rid of 2 liner checks everywhere then I suggest moving it to `LibraryTreeItem` and updating all calls of `ModelWidget::validateText` to `LibraryTreeItem::validateText`.

Reply:
Reverted it to the regular call sequence used everywhere else.